engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -1030,12 +1030,13 @@ namespace dmGameSystem
         return (dmGameObject::HComponent)params.m_UserData;
     }
 
+    // we can use only 24 bits, take a look render_private.h -> RenderListSortValue
     inline uint32_t MakeFinalRenderOrder(uint32_t world_order, uint32_t scene_order, uint32_t sub_order)
     {
         // scene_order: 4 bits (max 15)
-        // world_order: 12 bits (max 4095)
-        // sub_order:   16 bits (max 65535)
-        return (scene_order << 28) | (world_order << 16) | sub_order;
+        // world_order: 7 bits (max 127)
+        // sub_order:   13 bits (max 8191)
```

Comment:
It is basically draw calls count for all GUIs, I think it is enough to have 8192